target_redshift/db_sync.py

```diff
@@ -644,6 +644,7 @@ def load_csv(self, s3_key, count, size_bytes, compression=False, manifest=False)
                             FROM {stage_table} stage
                             JOIN {target_table} target
                             ON {join_condition}
+                            AND stage._sys_updated_at > target._sys_updated_at
```

Comment:
I think this will... work...

```
UPDATE {target_table}
 SET _sys_end_time = stage._sys_updated_at
 FROM {stage_table} stage
 JOIN {target_table} target
ON {join_condition}
AND stage._sys_updated_at > target._sys_updated_at
```

What I believe this will do is set the _sys_end_time=_sys_updated_at when the max(_sys_updated_at) in the stage table is strictly higher than the target _sys_updated_at. Then when we delete records it'll flush to history.